packages/client/katello-host-tools/katello-host-tools.spec

```
@@ -122,6 +122,7 @@ BuildArch:  noarch
 Summary:    Adds an fqdn fact plugin for subscription-manager
 Group:      Development/Languages
 
+Requires: %{name} = %{version}-%{release}
```

Nit: perhaps keep this aligned?
```suggestion
Requires:   %{name} = %{version}-%{release}
```